package-lock.json

```diff
@@ -328,8 +328,8 @@
       }
     },
     "node_modules/@bcrs-shared-components/enums": {
-      "version": "1.1.11",
-      "resolved": "https://registry.npmjs.org/@bcrs-shared-components/enums/-/enums-1.1.11.tgz",
+      "version": "1.1.12",
+      "resolved": "https://registry.npmjs.org/@bcrs-shared-components/enums/-/enums-1.1.12.tgz",
       "integrity": "sha512-/19lQ/DLo0hS0HWVNhhHqsjnp3Ed8f7g60kStQGFni6mOm1AHaEtSot9rUs56msYncmw4BNY6ho1o3ZvIeHlHQ==",
```

Comment:
This needed to be updated, too. Make sure you use npm to install/update packages. Don't just update the version manually.

Reply:
Resolved.